libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -9,6 +9,12 @@ $gd-header-height: 44px;
 $gd-header-animation-easing: cubic-bezier(0.23, 1, 0.32, 1);
 $gd-header-project-width: 230px;
 
+$button-border-hover: #27b3e0;
```

Comment:
If this variable can be replaced by `$gd-color-highlight`, then I think it's not necessary anymore. If that's right, please remove the variable.

Reply:
thank you we can use this for consistent with the styling of Gooddata. I just pick the color from modeler app to consistent wigth it but I think we should use gd-color-highlight and update in modeler later